tiboyce-convertsav/Makefile

```diff
@@ -0,0 +1,12 @@
+CC = gcc
+CFLAGS = -Wall -Wextra -O2
+TARGET = convertsav
```

Comment:
I'd prefer the target name be prefixed with `tiboyce-` to match with the Windows build and make the executable clearly identifiable